ros2_batch_job/__main__.py

```diff
@@ -61,47 +62,51 @@
 
 def main(sysargv=None):
     args = get_args(sysargv=sysargv)
-    blacklisted_package_names = [
-        'actionlib_msgs',
-        'shape_msgs',
-        'std_srvs',
-        'stereo_msgs',
-        'trajectory_msgs',
-        'visualization_msgs'
-    ]
-    return run(args, build_and_test, blacklisted_package_names=blacklisted_package_names)
-
-
-def get_args(sysargv=None, skip_white_space_in=False, skip_connext=False, add_ros1=False):
+    if not args.packaging:
+        blacklisted_package_names = [
+            'actionlib_msgs',
+            'shape_msgs',
+            'std_srvs',
+            'stereo_msgs',
+            'trajectory_msgs',
+            'visualization_msgs'
+        ]
+        return run(args, build_and_test, blacklisted_package_names=blacklisted_package_names)
+    else:
+        return run(args, build_and_test_and_package)
+
+
+def get_args(sysargv=None):
     parser = argparse.ArgumentParser(
         description="Builds the ROS2 repositories as a single batch job")
     parser.add_argument(
+        '--packaging', default=False, action='store_true',
+        help='Build and test the ros1_bridge as well as package the install space')
```

Comment:
Since testing of the ros1_bridge is decided by [another argument](https://github.com/ros2/ci/pull/33/files#diff-dcc9b88c6bfa21d6151296e463494a16R129) I would suggest removing it from this comment